zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/elasticsearch/ElasticsearchEngineClient.java

```diff
@@ -178,6 +184,34 @@ public void putIndexLifeCyclePolicy(final String policyName, final String deleti
     }
   }
 
+  @Override
+  public boolean importersCompleted(final int partitionId, final String indexPrefix) {
+    final var importPositionSearchRequest = importPositionDocuments(partitionId, indexPrefix);
+
+    try {
+      final var docs = client.search(importPositionSearchRequest, ImportPositionEntity.class);
+      return allImportersCompleted(
+          docs.hits().hits().stream().map(Hit::source).toList(), ImportValueType.values().length);
+    } catch (final IOException e) {
+      final var errMsg =
+          String.format(
+              "Failed to search documents in the import position index for partition [%s]",
+              partitionId);
+      LOG.error(errMsg, e);
+      throw new ElasticsearchExporterException(errMsg, e);
```

Comment:
❌ I think it is expectedto fail if this index doesn't exist? Then we should return true.

Reply:
The index should always exist either from the importer having created it, or the exporter creates it during the `open` phase.